en_US.po

```diff
@@ -1760,7 +1760,7 @@ msgid "管理标签"
 msgstr "Manage tags"
 
 msgid "文件"
-msgstr "Files."
+msgstr "Files"
 
 msgid "管理文件"
 msgstr "Manage files."
```

Comment:
I recommended also remove the trailing `.` of this

also,

```po
msgid "管理评论"
msgstr "Manage comments."
msgid "管理分类"
msgstr "Manage categories"
msgid "新增分类"
msgstr "New category."
```

and

```po
msgid "基本设置"
msgstr "Basic settings."
msgid "评论设置"
msgstr "Comments settings."
msgid "阅读"
msgstr "Read"
msgid "阅读设置"
msgstr "Reading settings."
```

https://github.com/typecho/languages/pull/44/files#diff-cfd107ff99de6153f55ecbdba3bde28fR1750-R1757
https://github.com/typecho/languages/pull/44/files#diff-cfd107ff99de6153f55ecbdba3bde28fR1780-R1790

Reply:
How about now? @pan93412